src/geode/mesh/core/tetrahedral_solid.cpp

```diff
@@ -325,6 +322,85 @@ namespace geode
         return result;
     }
 
+    template < index_t dimension >
+    absl::optional< PolyhedronFacet >
+        TetrahedralSolid< dimension >::polyhedron_adjacent_facet(
+            const PolyhedronFacet& polyhedron_facet ) const
+    {
+        const auto opt_polyhedron_adj =
+            this->polyhedron_adjacent( polyhedron_facet );
+        if( !opt_polyhedron_adj )
+        {
+            return absl::nullopt;
+        }
+        std::array< index_t, 3 > vertices;
```

Comment:
Why not calling polyhedron_facet_vertices()

Reply:
Same